services/cases-api/lambdas/updateCase.js

```diff
@@ -9,17 +13,44 @@ import { decodeToken } from '../../../libs/token';
 import { objectWithoutProperties } from '../../../libs/objects';
 import { getStatusByType } from '../../../libs/caseStatuses';
 
+import { getCaseWhereUserIsApplicant } from '../helpers/dynamo';
+import { updateCaseValidationSchema } from '../helpers/schema';
+
 export async function main(event) {
   const requestJsonBody = JSON.parse(event.body);
   const { id } = event.pathParameters;
+  const { personalNumber } = decodeToken(event);
 
   if (!id) {
-    return response.failure(new BadRequestError('missing [id] in query path'));
+    return response.failure(new BadRequestError('Missing required path parameter "id"'));
+  }
+
+  const { error, validatedEventBody } = updateCaseValidationSchema.validate(requestJsonBody, {
+    abortEarly: false,
+  });
```

Comment:
Incorrect use of `validate`. Should be:

```javascript
  const { error, value: validatedEventBody } = updateCaseValidationSchema.validate(
    requestJsonBody,
    {
      abortEarly: false,
    }
  );
```

Currently, `validatedEventBody` is always `undefined`.

Reply:
Might be something else that makes this act weird, since the documentation of validate clearly states the return value of the function to be an object with the attributes error and value. 

https://joi.dev/api/?v=17.4.0